### DISCUSSION TITLE
CSS pseudo-classes :nth-child(), :nth-of-type() and others

### DISCUSSION THREAD

**Discussion by AndreyGunderin** [acss-io/atomizer#525] [Q&A] [answered]
Good day, community!
Is it possible to choose some atomic class that provides such CSS pseudo-classes as :nth-child(), :nth-of-type() ?
Regards, Andrew.

**Comment by redonkulus** [comment, marked as answer]
We support [many pseudo-classes](https://github.com/acss-io/atomizer/blob/main/packages/atomizer/src/lib/grammar.js#L7), but not the `nth-child` and `nth-of-type`. The `:nth-child(odd)` and `:nth-child(even)` rules might be possible to add as helper classes but the more complex functional notation would be difficult to add with our current parsing logic.

**Reply by redonkulus** [reply]
@AndreyGunderin I actually played around with this and got something working, you can check it out here https://github.com/acss-io/atomizer/pull/527

**Reply by redonkulus** [reply]
I ended up closing the PR since just adding odd/even support is not that helpful. I hope to open a new PR when `:nth-child()` support can be added.
